src/main/java/mate/academy/hibernate/relations/util/HibernateUtil.java

```diff
@@ -1,9 +1,19 @@
 package mate.academy.hibernate.relations.util;
 
 import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
 
 public class HibernateUtil {
+    private static SessionFactory instance = initSessionFactory();
+
+    private HibernateUtil() {
+    }
+
     public static SessionFactory getSessionFactory() {
-        return null;
+        return new Configuration().configure().buildSessionFactory();
+    }
+
+    private static SessionFactory initSessionFactory() {
+        return instance;
     }
```

Comment:
you should change the implementations, init method should make this  
`return new Configuration().configure().buildSessionFactory();`
but the getSessionFactory() should only return existing